artwork/networks/network_portugal_logo.svg

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="25" height="64">
```

Comment:
Shouldn't it be `width="64"` here?

Reply:
Yes, sorry, I should have looked at it again :-/